api/manifest/manifest.go

```diff
@@ -1904,3 +1934,23 @@ func writeAsset(path, template string) error {
 
 	return writeFile(path, data, info.Mode())
 }
+
+func readDockerIgnore() ([]string, error) {
+	fd, err := os.Open(".dockerignore")
+
+	if os.IsNotExist(err) {
+		return []string{}, nil
+	}
+
```

Comment:
please aim for consistency in whitespace around error checking. No whitespace is preferred, which you do correctly above.